handlers/apps.go

```diff
@@ -59,7 +61,22 @@ func AppList(w http.ResponseWriter, r *http.Request) {
 	}
 
 	if criteria.IncludeHealth {
-		rateInterval, err := adjustRateInterval(r.Context(), business, p.Namespace, p.RateInterval, p.QueryTime)
+		// When the cluster is not specified, we need to get it. If there are more than one,
+		// get the one for which the namespace creation time is oldest
+		clusters, _ := business.Namespace.GetNamespaceClusters(r.Context(), p.Namespace)
```

Comment:
nit: these are `namespaces` not `clusters` right?

Reply:
That's right! Updated.